### PR TITLE
MC-136: Only used configurable attributes are now chosen to build configurable product

### PR DESCRIPTION
**Pull request by damien-carcel**
Bug fix: [yes]
Feature addition: [no]
Backwards compatibility break: [no]
Phpspec specification passes: [no]
Checkstyle issues: [no]
ChangeLog updated: [yes]
Fixes the following issue: Incorrect configurable attributes chosen in case not all used to build configurable product

### CODE REVIEW

File: Normalizer/ConfigurableNormalizer.php [hunk 1]
```diff
@@ -34,11 +34,10 @@ class ConfigurableNormalizer extends AbstractNormalizer
...
     /**
-     * Constructor
-     * @param ChannelManager      $channelManager
-     * @param ProductNormalizer   $productNormalizer
-     * @param PriceMappingManager $priceMappingManager
-     * @param boolean             $visibility
+     * @param ChannelManager       $channelManager
+     * @param ProductNormalizer    $productNormalizer
+     * @param PriceMappingManager  $priceMappingManager
+     * @param boolean              $visibility
```
Comment (fitn) [inline Normalizer/ConfigurableNormalizer.php]: not necessary

Comment (damien-carcel) [inline Normalizer/ConfigurableNormalizer.php]: Rha, I forget to remove the additional spacing after removing a parameter… I correct that immediately.

File: Normalizer/ConfigurableNormalizer.php [hunk 2]
```diff
@@ -194,8 +196,19 @@ protected function getDefaultConfigurable(
         $defaultProductValues[ProductNormalizer::VISIBILITY] = $this->visibility;
         $defaultProductValues[ProductNormalizer::URL_KEY] = $defaultProductValues[ProductNormalizer::URL_KEY].'-conf-'.$group->getId();
 
+        $configurableAttributes = [];
```
Comment (fitn) [inline Normalizer/ConfigurableNormalizer.php]: Useless?

Comment (damien-carcel) [inline Normalizer/ConfigurableNormalizer.php]: Hem, indeed…

**Pull request merged**